lib/bundler/settings.rb

```diff
@@ -41,6 +41,7 @@ class Settings
       ignore_messages
       init_gems_rb
       list_command
+      lockfile_upgrade_warning
```

Comment:
@colby-swandale You know what this setting (only currently present on 2-0-stable) is about? Is it related to the whole discussion about whether to print warnings or errors or warnings on version mismatches?

Reply:
Oh i missed this. So when i was working on the Bundler 2 release, i was getting a whole bunch of failing tests because of the warning that this feature flag is wrapping.


I'm not sure this is still needed or not. I'll check and see if we're still having issues.

Reply:
Specs seem to pass without it now. I'm going to remove it in this branch and then make a PR to remove it in `2-0-stable`